Libraries/xcexecution/Sources/SimpleExecutor.cpp

```diff
@@ -258,7 +257,8 @@ performInvocations(
                         processContext->groupID(),
                         processContext->userName(),
                         processContext->groupName());
-                    success = driver->run(&context, filesystem) != 0;
+                    int exitCode = driver->run(&context, filesystem);
+                    success = (exitCode == 0);
```

Comment:
Made it a little more obvious that it's returning an `int` rather than a `bool`. I already missed the `!= 0` once when it was on the same line. 😶